### PR TITLE
Fix build after new debugger-libs submodule

### PR DESCRIPTION
**Pull request by akoeplinger**
Bump to new commit that doesn't use packages.config anymore.

Also copy files from the correct location so the build works

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -180,24 +183,26 @@ $(addprefix bin/, $(deps)):
...
-	$(CP) dep/debugger-libs/Mono.Debugging.Soft/bin/Debug/Mono.Debugging.Soft.dll \
+	$(CP) dep/debugger-libs/Mono.Debugging.Soft/bin/$(MODE)/Mono.Debugging.Soft.dll \
 		bin/Mono.Debugging.Soft.dll
-	$(CP) dep/debugger-libs/packages/System.Collections.Immutable.1.3.1/lib/netstandard1.0/System.Collections.Immutable.dll \
+	$(CP) dep/debugger-libs/Mono.Debugging.Soft/bin/$(MODE)/System.Collections.Immutable.dll \
 		bin/System.Collections.Immutable.dll
-	$(CP) dep/debugger-libs/packages/System.Reflection.Metadata.1.4.2/lib/netstandard1.1/System.Reflection.Metadata.dll \
+	$(CP) dep/debugger-libs/Mono.Debugging.Soft/bin/$(MODE)/System.Reflection.Metadata.dll \
 		bin/System.Reflection.Metadata.dll
+	$(CP) dep/debugger-libs/Mono.Debugging.Soft/bin/$(MODE)/Newtonsoft.Json.dll \
```
Comment (alexrp) [inline Makefile]: minor point: would be nice to keep the commands sorted

**Comment by akoeplinger** [comment]
Should be ready now :)

**Comment by akoeplinger** [comment]
@alexrp can this be merged?

**Comment by alexrp** [comment]
Sure, looks good to me.

But I seem to have lost write access to this repository at some point, so you'll need to press the merge button yourself.

**Comment by akoeplinger** [comment]
lol ok, thanks. let me know if you want it back and I can make that happen.

**Pull request merged**